src/fairchem/core/common/logger.py

```diff
@@ -92,8 +94,8 @@ def __init__(self, config) -> None:
             group=group,
         )
 
-    def watch(self, model, log_freq: int = 1000) -> None:
-        wandb.watch(model, log_freq=log_freq)
+    def watch(self, model, log="all", log_freq: int = 1000) -> None:
```

Comment:
Just checking this change was meant to log by default or hardcoded in mt trainer , both gradients and parameters? its just 2x to include parameters, just checking it was intended

Reply:
ya, it was not logging everything previously